lib/switchtec.c

```diff
@@ -228,6 +247,9 @@ struct switchtec_dev *switchtec_open(const char *device)
 	if (set_gen_variant(ret))
 		return NULL;
 
+	if (set_local_pax_id(ret))
```

Comment:
This is going to break any hardware that does not support MRPC_GET_PAX_ID.

Reply:
Should return 1 on non-pax switch, will update.

Reply:
Ooh, you mean the MRPC_GET_PAX_ID. I think, for PAX, the firmware need to have a way to report the pax id of the locally connected PAX switch, otherwise, we need to quit. It's like we cannot get the partition of a PSX switch.